cob_cartesian_controller/src/trajectory_interpolator/trajectory_interpolator.cpp

```diff
@@ -73,18 +79,9 @@ bool TrajectoryInterpolator::linearInterpolation(geometry_msgs::PoseArray& pose_
     // Interpolate the linear path
     for(int i = 0 ; i < path_matrix[0].size() ; i++)
     {
-        if(move_lin.rotate_only)
```

Comment:
so is ```rotate_only``` obsolete now?
if yes please remove from data_structures, classes and message definitions....